public/js/clients/chrome/InspectorBackend.js

```diff
@@ -0,0 +1,704 @@
+/**
```

Comment:
EDIT: I see the another file has a "comes from" message, but it's just a relative path that doesn't link to anywhere. I haven't finished this PR yet, but it might be nice to have a `.md` in here describing exactly where to get these files.